pkgs/development/libraries/liberfa/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "erfa";
+  version = "1.7.0";
+
+  buildInputs = [ autoreconfHook ];
+
+  src = fetchFromGitHub {
+    owner = "liberfa";
+    repo = "erfa";
+    rev = "v${version}";
+    sha256 = "1z4k2phrw6wwi0kax6ac80jk9c036gi7pmhmg6gaf3lk81k6xz2r";
+  };
+
+  configureFlags = [ "--enable-shared" ];
+
+  meta = with stdenv.lib; {
+    description = "A C library containing key algorithms for astronomy, and is based on the SOFA library published by the International Astronomical Union (IAU).";
+    homepage = "https://github.com/liberfa/erfa";
+    license = "https://github.com/liberfa/erfa/blob/master/LICENSE";
```

Comment:
`license` should not be a link to the license file. Again, see https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes.

Reply:
Well, then it shall be `free` as far as I can judge ERFA's license, right?

Reply:
@timokau that license doesn't seem "standard", or like one of those available in https://github.com/NixOS/nixpkgs/blob/master/lib/licenses.nix . The manual says:

> [..] or in-place license description of the same format if the license is unlikely to be useful in another expression. 

Though I don't see a way to specify a "description" of _this_ license - it starts with a `Copyright (C) 2013-2014, NumFOCUS Foundation.` but after reading more it feels like it's a Free software...

I did a `git grep 'license =' | grep LICENSE` and there are some derivations that use a straight link to upstream's license, and some use `license.free` along with a comment to upstream's license file. Personally I like the approach of a straight link as it's parseble by Nix.

Reply:
@doronbehar technically parsable with nix yes, but a URL doesn't provide much practical value when parsed with nix. `free` would actually be more valuable, since you could at least sort packages on whether they're free/unfree.

I think with "of the same format" it is meant that you should just define a license attrset inline. That could for example be modeled after this entry in `licenses.nix`:

```nix
  epson = {
    fullName = "Seiko Epson Corporation Software License Agreement for Linux";
    url = "https://download.ebz.epson.net/dsc/du/02/eula/global/LINUX_EN.html";
    free = false;
  };
```